source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -4773,6 +4796,10 @@ export abstract class TypeRegistry<T> {
     getItemAt(index: number): ITypeRegistration<T> {
         return this._items[index];
     }
+
+    getItems(): ITypeRegistration<T>[] {
```

Comment:
Please remove, this isn't necessary and the `_items` field isn't meant to be directly accessible.

Reply:
Already in progress

Reply:
(use getitemcount, getitemat)

Reply:
In progress.